http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -102,7 +102,7 @@ export JUPYTER_SERVER_NAME=$(jupyterServerName)
 export JUPYTER_DOCKER_IMAGE=$(jupyterDockerImage)
 export JUPYTER_START_USER_SCRIPT_URI=$(jupyterStartUserScriptUri)
 # Include a timestamp suffix to differentiate different startup logs across restarts.
-export JUPYTER_START_USER_SCRIPT_OUTPUT_URI="$(jupyterStartUserScriptOutputBaseUri)-$(date -u "+%Y.%m.%d-%H.%M.%S").txt"
+export JUPYTER_START_USER_SCRIPT_OUTPUT_URI=$(jupyterStartUserScriptOutputUri)
```

Comment:
Hm - do we need a timestamp per this comment? AoU might be consuming these logs.

Reply:
I see we are using GCS object metadata to check on the result, so want a consistent output URI.

What if we wrote 2 files every time:
- `$jupyterStartUserScriptOutputUri` -- always contains the 'latest' output, can be used to store "passed" metadata flag
- `$jupyterStartUserScriptOutputUri-$(timestamp)` -- tracks output over time

Another idea is to use GCS object versioning.

Reply:
Oh wait.. we're still appending the timestamp, just in `RuntimeTemplateValues` instead of here. Do I have that right?

I do think `startup.sh` needs to be updated then, in addition to this script.

Reply:
yea...I'm moving this timestamp out of the script so that leo knows about which file to look up later